src/Tribe/Importer/File_Importer.php

```diff
@@ -62,6 +63,11 @@ public static function get_importer( $type, Tribe__Events__Importer__File_Reader
 	public function __construct( Tribe__Events__Importer__File_Reader $file_reader, Tribe__Events__Importer__Featured_Image_Uploader $featured_image_uploader = null ) {
 		$this->reader = $file_reader;
 		$this->featured_image_uploader = $featured_image_uploader;
+
+		//Add Vendor Directory for Autoloading for Encoding Class
+		$autoloader = Tribe__Autoloader::instance();
+		$autoloader->register_prefix( 'Vendor__', Tribe__Events__Main::instance()->plugin_path . 'vendor' );
+		$autoloader->register_autoloader();
```

Comment:
This works, but I think I'd be inclined to move it to `Tribe__Events__Main::init_autoloading()` (and add it to the array of prefixes being registered there), reason being if we do find some other place we want to use the UTF8 class in the future - outside of the importer code - that will prevent us from having to re-register the same prefix there.

One other thing is I'd probably stick with Matt's original suggestion of naming it `ForceUTF8__Encoding` - so the prefix being registered would be `ForceUTF8__` rather than `Vendor__` and the path for that prefix would become `vendor/ForceUTF8`, if that makes sense.